docs/user-roles.rst

```diff
@@ -29,7 +29,7 @@ CrateDB Cloud user roles
 
 This section describes the roles that can be set for users within CrateDB
 Cloud. For information on how to do so, see our documentation on ref:`adding
-users <crate-reference:add-users>`.
+users <crate:administration_user_management>`.
```

Comment:
According to [1], the prefix is really actually `crate-reference`?

[1] https://github.com/crate/crate-docs-theme/blob/0.15.3/src/crate/theme/rtd/conf/__init__.py#L75

Reply:
How would the tests succeed in this case?

Reply:
If you look at the actual document as it is now, you'll see the link is not a functional link: https://crate.io/docs/cloud/reference/en/latest/user-roles.html#cratedb-cloud-user-roles

As far as I can tell the actual repo is crate/docs and the relevant Sphinx header is administration_user_management?

Reply:
Oh it looks like the problem is the :ref: link is lacking a colon 😳